AlphaWallet/Transfer/Controllers/TransactionConfigurator.swift

```diff
@@ -198,11 +198,11 @@ class TransactionConfigurator {
                 completion(.failure(AnyError(Web3Error(description: "malformed tx"))))
             }
 
-        case .ERC721Token:
+        case .ERC721Token(let token):
             do {
                 let function: Function
                 let parameters: [Any]
-                if transaction.to!.isLegacy721Contract {
```

Comment:
@hboon this is the recipient address which is wrong

Reply:
So bizzarre.